invenio/modules/access/control.py

```diff
@@ -1804,11 +1804,11 @@ def acc_delete_all_settings():
     from invenio.ext.sqlalchemy import db
     db.session.commit()
 
-    run_sql("""TRUNCATE "accROLE" """)
-    run_sql("""TRUNCATE "accACTION" """)
-    run_sql("""TRUNCATE "accARGUMENT" """)
-    run_sql("""TRUNCATE "user_accROLE" """)
-    run_sql("""TRUNCATE "accROLE_accACTION_accARGUMENT" """)
+    truncate_table("accROLE")
```

Comment:
Can you use the model if you are changing `run_sql` in `invenio.modules.*`? Thanks